### PR TITLE
add favorite trading pair

### PR DESCRIPTION
**Pull request by AntonNazarenko**
Added ability to choose your favorite pairs
After a user selected the favorite pairs they will be shown in the Favorite Trading Pairs section
Also a user able to delete a favorite trading pair just by clicking on the star

### CODE REVIEW

**Comment by avsek477** [comment]
lgtm

File: src/ui/Select/Select.js [hunk 1]
```diff
@@ -1,24 +1,58 @@
 import React from 'react'
 import ClassNames from 'classnames'
 import RSelect from 'react-select'
-
+import FavoriteIcon from '../Icons/FavoriteIcon'
 import { propTypes, defaultProps } from './Select.props'
 import './style.css'
 
+const customOption = ({ innerProps, children, data }, onFavoriteSelect, favoritePairs) => {
```
Comment (robertkowalski) [inline src/ui/Select/Select.js:8]:
```
Failed to compile.

./src/ui/Select/Select.js
  Line 8:25:  'innerProps' is missing in props validation  react/prop-types
  Line 8:37:  'children' is missing in props validation    react/prop-types
  Line 8:47:  'data' is missing in props validation        react/prop-types
  Line 9:11:  'data.value' is missing in props validation  react/prop-types

Search for the keywords to learn more about each error.
```

Comment (robertkowalski) [reply src/ui/Select/Select.js:8]: read the thread with abishekd, maybe try to rebase with current master

File: src/components/FavoriteTradingPairs/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+  render() {
+    const { selectedPair } = this.state
+    const { favoritePairs = [] } = this.props
+    const hasFavoritePairs = favoritePairs.length !== 0
+    return (
+      <div className='hfui-favoritepair__wrapper'>
+        <div className='hfui-favoritepair__title'>FAVORITE TRADING PAIRS</div>
+        { !hasFavoritePairs && (
+          <div className='hfui-favoritepair__content empty'>Empty</div>
```
Comment (robertkowalski) [inline src/components/FavoriteTradingPairs/index.js:46]: better: `No Favorites starred yet`

**Review by robertkowalski — CHANGES_REQUESTED** [review]
nice work! after fixing the linting errors we can merge.

note: merge requires a package bump for bfx-api-models

**Comment by robertkowalski** [comment]
linter is happy now, can you fix https://github.com/bitfinexcom/bfx-hf-ui/pull/266#discussion_r559496647 with the improved wording? thanks!

**Review by robertkowalski — CHANGES_REQUESTED** [review]
linter is happy now, can you fix the remaining issue with the improved wording?

https://github.com/bitfinexcom/bfx-hf-ui/pull/266/files#r559496647

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**